portfolio.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+ <head>
+	 <link rel="stylesheet" type="text/css" href="normalize.css">
+	 <link rel="stylesheet" type="text/css" href="portfolio.css"  >
+	 <title> Portfolio Simona Ambrosini</title>
+ </head>
+ 
+	
+ <body><
+	 <nav>
+		 <button>Menu</button>
+		 <div class="links">
+	       <a href="navbar.asp" style="text-decoration:none">Home</a>
+	 	   <a href="about.asp" style="text-decoration:none">About</a>
+	 	   <a href="portfolio.asp" style="text-decoration:none">Portfolio</a>
+	 	   <a href="contact.asp" style="text-decoration:none">Contact me</a>
+	 	</div>
+		<div class="download">
+			<a href="|">Download CV</a>
+		</div>
+	 </nav>
+	 <header>
+		 <div class="intro">
+		 <h1>Simona Ambrosini</h1>
+		 <h2></h2>
+	 </div>
```

Comment:
I think this should be `</header>` in which case you can remove the `<div>` above. If you want to style the header like a div you can set a class="intro" attribute there instead